jekyll/_cci2/release/set-up-a-release-environment.adoc

```diff
@@ -89,16 +89,20 @@ helm repo update
 [#install-the-release-agent]
 == 3. Install the release agent
 
-Install the Helm chart to integrate CircleCI's release agent into your Kubernetes cluster. Be sure to substitute the token you created in the previous step, and also include a list of Kubernetes namespaces you want to manage:
+Install the Helm chart to integrate CircleCI's release agent into your Kubernetes cluster. Be sure to substitute the token you created in the previous step, and also, if required, include a list of Kubernetes namespaces you want to monitor.
+
+If you only want to monitor the `default` namespace, you do not need to supply a list with `--set manageNamespaces=[YOUR_LIST_OF_NAMESPACES_TO_MONITOR]`. The `default` namespace is monitored by default.
 
 [,shell]
 ----
 helm upgrade --install circleci-release-agent-system release-agent/circleci-release-agent \
 --set tokenSecret.token=[YOUR_CCI_INTEGRATION_TOKEN] --create-namespace \
 --namespace circleci-release-agent-system \
---set manageNamespaces=[YOUR_LIST_OF_NAMESPACES_TO_MANAGE]
+--set manageNamespaces=[YOUR_LIST_OF_NAMESPACES_TO_MONITOR]
```

Comment:
using arrays in helm commands is super tricky, can we use this example?

`--set manageNamespaces={namespace1,namespace2}`

Reply:
Oh yes good call, thanks

Reply:
in addition, if it is not too much we can say

protip: if you struggle to pass the `manageNamespaces` value try enclosing the array value or the whole parameter itself in a string, with double or single quotes.